docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,270 @@
+
+Additionally, by default one agent is elected as **leader**(for more information visit <<kubernetes_leaderelection-provider>>). This is configured in our https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L32[manifests]. The Agent Pod which holds the leadership lock will be responsible for also collecting the cluster wide metrics in addition to its node's metrics.
+
+--
+[role="screenshot"]
+image::../images/k8sscaling.png[Agent as daemonset]
+--
+
+The above schema explains how {agent} collects and sends metrics to Elasticsearch. Because of Leader Agent being responsible to also collecting cluster-lever metrics, this means that it requires additional resources.
+
+Daemonset deployment approach simplifies the installation of the {agent} because we define less Kubernetes Resources in our manifest and we only need one single Agent policy for our Agents. Hence it is the only supported way of <<running-on-kubernetes-managed-by-fleet, managed Elastic Agent scenario>>
+
+
+[discrete]
+=== Specifying resources and limits in Agent manifests
+
+Based on our tests we advise to configure only the `limit` section of the `resources` section in the manifest. In this way the `request`'s settings of the `resources` will fall back to the `limits` specified. The `limits` is the upper bound limit of your microservice process, meaning that can operate in less resources and protect Kubernetes to assign bigger usage and protect from possible resource exhaustion.
+
+[source,yaml]
+------------------------------------------------
+resources:
+    limits:
+      cpu: "1000m"
+      memory: "200Mi"
+------------------------------------------------
+
+
+Based on our [tests](https://github.com/elastic/elastic-agent/blob/7467fd72bccab9a0e1b0adc1761cde8520390943/docs/elastic-agent-scaling-tests.md), the following table provides guidelines to adjust {agent} limits on different kubernetes sizes:
+
+Sampe Elastic Agent Configurations:
+|===
+| No of Pods in K8s Cluster | Leader Agent Resources | Rest of Agents 
+| 1000   | cpu: "1500m",  memory: "800Mi" | cpu: "300m",  memory: "600Mi" 
+| 3000   | cpu: "2000m",  memory: "1000Mi" | cpu: "400m",  memory: "800Mi" 
+| 5000   | cpu: "3000m",  memory: "2500Mi" | cpu: "500m",  memory: "900Mi" 
+| 10000  | cpu: "3400m",  memory: "4500Mi" | cpu: "700m",  memory: "1000Mi" 
+|===
+
+> Above tests run with {agent} version 8.7 and scraping period of 10sec in Kubernetes Policy. Those numbers are just indicators and should be validated per different kubernetes policy configuration, along with applications that the kubernetes cluster might include
+
+[discrete]
+=== Proposed Agent Installations for large scale
+
+Although daemonset installation is simpler, it can not diffrentiate the alternative agent resource requirements depending on the collected metrics. The need for better 
```

Comment:
```suggestion
Although daemonset installation is simpler, it can not differentiate the alternative agent resource requirements depending on the collected metrics. The need for better 
```